goci-interfaces/goci-ui/src/main/resources/static/js/studyresult.js

```diff
@@ -157,8 +157,13 @@ function displayUnpublishedStudySummary(accession) {
         $("#genotyping-tech").html(data.genotyping_technology);
         $("#date-submitted").html(new Date(data.createdDate).toDateString());
         $("#trait").html(data.efo_trait);
-        $("#platform-snp").html('');
-        $("#preprint-doi").attr('href', data.body_of_work[0].doi);
+        var imputation = data.imputation ? ' (imputed)' : '';
+        $("#platform-snp").html(data.array_manufacturer + ' [' + data.variant_count + ']' + imputation)
+        if (data.body_of_work[0].doi != null) {
+            var preprintLink = '<a href="' + data.body_of_work[0].doi + '" target="_blank"> View Preprint </a>'
```

Comment:
HI @ala-ebi I think it'l be better if this line can just be implemented as a template literal,

$("#preprint-doi").html(`<a href="${data.body_of_work[0].doi}" target="_blank"> View Preprint </a>`)

 I think its cleaner and more readable than the plus.